src/number_theory/bernoulli.lean

```diff
@@ -204,3 +203,37 @@ begin
   { apply factorial_mul_factorial_dvd_factorial_add, },
   { exact cast_ne_zero.mpr hj', },
 end
+
+open ring_hom
+
+/-- Odd Bernoulli numbers (greater than 1) are zero. -/
+theorem bernoulli_odd_eq_zero {n : ℕ} (h_odd : n % 2 = 1) (hlt : 1 < n) : bernoulli n = 0 :=
```

Comment:
There are dedicated predicates `odd` and `even`. It is better to use those in the assumption.
There should already be a lemma that moves between `odd n` and `n % 2 = 1`.
So in the worst case, your proof gets one line longer, and in the best case, the api for `odd` helps you simplify some steps of the proof.

Reply:
Done! One less line of code now :)